NumLib/DOF/LocalToGlobalIndexMap.cpp

```diff
@@ -191,4 +191,24 @@ std::ostream& operator<<(std::ostream& os, LocalToGlobalIndexMap const& map)
 }
 #endif  // NDEBUG
 
+NumLib::LocalToGlobalIndexMap::RowColumnIndices getRowColumnIndices(
```

Comment:
There already is a very similar method (`getIndices()`) in NumLib/DOF/DOFTableUtil.cpp
Your `getVectorValues()` is already provided by `EigenVector::get()` etc.
As far as I can see.